libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -187,13 +216,15 @@ void CodeTransform::operator()(assembly::Identifier const& _identifier)
 		}
 	)))
 	{
+		decreaseReference(_identifier.name);
 		return;
 	}
 	solAssert(
 		m_identifierAccess.generateCode,
 		"Identifier not found and no external access available."
 	);
 	m_identifierAccess.generateCode(_identifier, IdentifierContext::RValue, m_assembly);
+	decreaseReference(_identifier.name);
```

Comment:
Ah I'm sorry, this is wrong. I think the only place where `decreaseReference` should be called is the one in line 220 above. Especially, there should be no call to `decreaseReference` in this line, because this line handles references to variables outside of the scope of assembly.